app.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ah-code-blooded",
+  "scripts": {},
+  "env": {
+    "DISABLE_COLLECTSTATIC": {
```

Comment:
Why `DISABLE_COLLECTSTATIC` ??

Reply:
I'm disabled it since the API is not serving static content.

Reply:
After some consideration, I think it needs to serve static content for the browsable API.